examples/sklearn/demo_new_features.ipynb

```diff
@@ -0,0 +1,782 @@
+{
```

Comment:
Regarding the last question: since the data is just a DataFrame now, the easiest way would probably be to add a new level to the index (e.g. `df.set_index([...], append=True)`). I could add an example to the notebook too.